main.py

```diff
@@ -1,38 +1,44 @@
 import argparse
```

Comment:
redesign all main.py file
add main() method and its call in correct way https://docs.python.org/3/library/__main__.html